lib/application.js

```diff
@@ -291,10 +291,27 @@ app.dataSources = app.datasources = {};
  * Enable app wide authentication.
  */
 
-app.enableAuth = function() {
+app.enableAuth = function(options) {
+  var AUTH_MODELS = ['User', 'AccessToken', 'ACL', 'Role', 'RoleMapping'];
```

Comment:
That's a good question. This shortcut is intended only for the simple situations where you typically don't create subclasses. To keep the implementation simple, I can modify the condition determining whether a base model should be attached or not, and do not attach a base model if there is a subclassed model already attached?

From user perspective, if you create let's say Customer extending User:

 - If Customer is attached to a data source or the app, then the builtin User model is left unchanged.
 - If Customer is not attached to the app nor the datasource, then we attach the builtin User model for you.

Thoughts?